<ui version="4.0" >
 <class>SignalAdaptorKeyFrameValue</class>
 <widget class="QWidget" name="SignalAdaptorKeyFrameValue" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>69</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>1</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="2" >
    <widget class="QToolButton" name="maxButton" >
     <property name="text" >
      <string>max</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="minButton" >
     <property name="text" >
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLineEdit" name="lineEdit" />
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QComboBox" name="comboBox" />
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QCheckBox" name="checkBox" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
